File: python/pembeli.py
class Pembeli:
    def __init__(self, id_pembeli, nama, alamat, kontak):
        self.id_pembeli = id_pembeli
        self.nama = nama
        self.alamat = alamat
        self.kontak = kontak

    # Setter
    def set_nama(self, nama):
        self.nama = nama

    def set_alamat(self, alamat):
        self.alamat = alamat

    def set_kontak(self, kontak):
        self.kontak = kontak

    # Getter
    def get_id(self):
        return self.id_pembeli

    def get_nama(self):
        return self.nama

    # Display info
    def display_info(self):
        print("+-----------------------------+")
        print("|        INFORMASI PEMBELI    |")
        print("+-----------------------------+")
        print(f"| Nama   : {self.nama}")
        print(f"| Alamat : {self.alamat}")
        print(f"| Kontak : {self.kontak}")
        print("+-----------------------------+\n")

File: php/pembeli.php
<?php
class Pembeli {
    // atribut
    private $id;
    private $nama;
    private $alamat;
    private $telp;
    private $gambar;

    // constructor
    public function __construct($id, $nama, $alamat, $telp, $gambar) {
        $this->id = $id;
        $this->nama = $nama;
        $this->alamat = $alamat;
        $this->telp = $telp;
        $this->gambar = $gambar;
    }

    // GETTER
    public function getId() { return $this->id; }
    public function getNama() { return $this->nama; }
    public function getAlamat() { return $this->alamat; }
    public function getTelp() { return $this->telp; }
    public function getGambar() { return $this->gambar; }

    // SETTER
    public function setNama($nama) { $this->nama = $nama; }
    public function setAlamat($alamat) { $this->alamat = $alamat; }
    public function setTelp($telp) { $this->telp = $telp; }
    public function setGambar($gambar) { $this->gambar = $gambar; }
}
